.jenkins/scripts/release/release-3-perform.sh

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+#
+#  Copyright (C) 2006-2024 Talend Inc. - www.talend.com
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#
+
+set -xe
+
+# Parameters:
+# $1: extra build args for all mvn cmd
+main() {
+  local extraBuildParams=("$@")
+
+  printf ">> Maven perform release"
+
+  local release_profiles="release,ossrh,gpg2"
+
+  mvn release:perform \
+    help:effective-pom \
+    --define output="effective-pom-prepare.xml" \
```

Comment:
Why generate effective poms in production?

Reply:
For the effective pom, I was thinking it wouldn't cost much to leave them for the moment to debug if we have an issue, but I can remove them if you prefer.

Reply:
removed